apps/fluent-tester/src/FluentTester/theme/applyTheme.ts

```diff
@@ -56,7 +56,7 @@ export function applyTheme(parent: Theme, name: ThemeNames): PartialTheme {
   switch (name) {
     case 'Office':
       return themingModule
-      ? createOfficeTheme({ paletteName: 'WhiteColors' }).theme
+      ? createOfficeTheme({ paletteName: 'Menus_FluentSV' }).theme
```

Comment:
Please don't commit this change - the copy of Office dlls we have is old and doesn't include this palette, so we will hit an error when running the tester outside of devmain.

Reply:
Yeah, I wanted to clarify this :) Will remove